argo-events.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: argo-events
+  version: 1.9.2
+  epoch: 0
+  description: Event-driven Automation Framework for Kubernetes.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 8dd63f1305746e6503269a7675288b1f65e53df5
+      repository: https://github.com/argoproj/argo-events
+      tag: v${{package.version}}
+
+  - uses: go/build
+    with:
+      ldflags: |
+        -s -w
+        -X github.com/argoproj/argo-events.version=${{package.version}}
+        -X github.com/argoproj/argo-events.gitCommit=$(git rev-parse --short=8 HEAD)
+        -X github.com/argoproj/argo-events.buildDate=$(date -u -d "@${SOURCE_DATE_EPOCH:-$(date +%s)}" "+%Y-%m-%dT%H:%M:%SZ")
+        -X github.com/argoproj/argo-events.gitTreeState=clean
+      output: argo-events
+      packages: ./cmd
+
+  - uses: strip
```

Comment:
Strip is handled via `-w` in ldflags within `go/build`

Reply:
I was using the same approach as the other argo package. Should we start removing this from the other packages as well to make it all equal? this can confuse people like me

Reply:
thanks for the review !!